R/DittoScatterPlot.R

```diff
@@ -229,6 +232,8 @@ dittoScatterPlot <- function(
     legend.color.breaks.labels = waiver(),
     legend.shape.title = shape.by,
     legend.shape.size = 5,
+    raster = FALSE,
```

Comment:
I'd like this to be `do.raster` in order to match with the naming convention for other added features.